src/run_releasenotes_check.py

```diff
@@ -71,15 +88,17 @@ def create_urls_file_if_not_exists() -> None:
 
         urls_filepath = os.path.join(os.path.dirname(__file__), urls_filename)
         os.makedirs(os.path.dirname(urls_filepath), exist_ok=True)
-        with open(urls_filepath, "w") as urls_file:
+        with open(urls_filepath, "a") as urls_file:
```

Comment:
Benefit of changing from w to a?

Reply:
Previously we were processing only one manifest, now since we process more than one and process them one by one in a loop we need to append, as `w` will overwrite the contents of the previous manifest.

Reply:
Got it!